usagov/usagov-2021
# USAGOV-1421: Correct the path to the S3 backups for public files

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1421

## Description

Corrected the path to the snapshot backups for public files in the downsync-cms script.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


I ran this (from this branch) after the next prod deployment to verify. Files don't actually change that often, but we can tell whether the files downsynched by looking at the contents of the directory the script downloads public files into (it will be [snapshot-tag]_files). This script has been generating an empty directory. With the fix, it should contain files. 

Proof of files: 
![Screenshot 2024-02-27 at 6 39 40 PM](https://github.com/usagov/usagov-2021/assets/35591/cb5823af-96f8-43ac-b9a8-0bbd6b0b8646)


### Requires New Config

### Requires New Content

### Validation Steps
- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
